examples/whylogs_data_profiling/steps/profiler/profiler_step.py

```diff
@@ -11,16 +11,25 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 #  or implied. See the License for the specific language governing
 #  permissions and limitations under the License.
-from zenml.integrations.whylogs.steps import whylogs_profiler_step
+from zenml.integrations.whylogs.steps import (
+    WhylogsProfilerConfig,
+    whylogs_profiler_step,
+)
 
-# Another quick way of enhancing your pipeline with whylogs profiling features
+# A quick way of enhancing your pipeline with whylogs profiling features
 # is with the `whylogs_profiler_step` function, which creates a step that runs
 # whylogs data profiling on an input dataframe and returns the generated
 # profile as an output artifact.
 
 train_data_profiler = whylogs_profiler_step(
-    "train_data_profiler", dataset_name="train", tags={"datasetId": "model-15"}
+    step_name="train_data_profiler",
+    config=WhylogsProfilerConfig(),
+    log_to_whylabs=True,
```

Comment:
why would you set up a whylogs profiler step and *not* log to whylabs?

Reply:
you make a good point :D I'll remove this and assume it's always True